tests/beta/test_default_tensor.py

```diff
@@ -695,24 +864,29 @@ def test_unsupported_observable_error(self, tensornet_device_3_wires, observable
         else:
             wires = list(range(op.num_wires))
 
-        @qml.qnode(tensornet_device_3_wires)
+        dev = qml.device("default.tensor", wires=3, representation=rep)
+
+        @qml.qnode(dev)
         def circuit(*x):
             """Test quantum function"""
             x = prep_par(x, op)
             return qml.expval(op(*x, wires=wires))
 
         with pytest.raises(
-            QuantumFunctionError, match="Device default.tensor is a qubit device; CV operations are not allowed."
+            QuantumFunctionError,
+            match="Device default.tensor is a qubit device; CV operations are not allowed.",
         ):
             x = np.random.random([op.num_params])
             circuit(*x)
 
-    def test_qubit_circuit(self, tensornet_device_1_wire, tol):
+    def test_qubit_circuit(self, rep, tol):
         """Test that the tensor network plugin provides correct result for a simple circuit"""
 
         p = 0.543
 
-        @qml.qnode(tensornet_device_1_wire)
+        dev = qml.device("default.tensor", wires=1, representation=rep)
```

Comment:
Really, really glad those fixtures have been removed. I'm a big fan of tests being self contained where possible.

If you can almost copy and paste an integration test into a python console and run it with minimal modification, that is 👌 in my books

Reply:
that's literally the reason these got deleted. I was trying to run them locally when debugging